index.html

```diff
@@ -104,12 +107,15 @@ <h2 class="notice__title">Ваше объявление</h2>
         </fieldset>
         <fieldset class="ad-form__element">
           <label class="ad-form__label" for="price">Цена за ночь, руб.</label>
-          <input id="price" name="price" type="number" placeholder="5000">
+          <input id="price" type="number" name="price" placeholder="5000" min="0" max="100000" required
+          data-pristine-required-message="Обязательное поле"
+          data-pristine-min-message="Минимальная цена 0 руб."
+          data-pristine-max-message="Максимальная цена 100000 руб.">
```

Comment:
Когда атрибутов много следует их располагать в столбик, ты правильно сделала для атрибутов пристин, только необходимо все так перенести
```suggestion
          <input
            id="price"
            type="number"
            ....
          >
```